### DISCUSSION TITLE
When 1.18.2 Forge?

### DISCUSSION THREAD

**Discussion by MrChallenge** [terrarium-earth/Ad-Astra#315] [Ideas]
I need ad astra on 1.18.2 Forge

**Comment by CodexAdrian** [comment]
Most likely never

**Reply by MrChallenge** [reply]
why?

**Reply by CodexAdrian** [reply]
> why?

We just decided it wasn’t worth it for us.

**Comment by Ureiah** [comment]
It costs more resources and time to backport to an older version. We only have a few developers, so we can't afford to backport. Sorry about that!

**Reply by MrChallenge** [reply]
Can other devs help?
